internal/collector/pgadmin.go

```diff
@@ -56,10 +60,32 @@ func EnablePgAdminLogging(ctx context.Context, configmap *corev1.ConfigMap) erro
 			{
 				"context": "log",
 				"statements": []string{
+					// Set instrumentation scope
+					`set(instrumentation_scope.name, "pgadmin")`,
+
+					// Keep the unparsed log record in a standard attribute, and replace
+					// the log record body with the message field.
+					//
+					// https://github.com/open-telemetry/semantic-conventions/blob/v1.29.0/docs/general/logs.md
+					`set(attributes["log.record.original"], body)`,
 					`set(cache, ParseJSON(body))`,
 					`merge_maps(attributes, ExtractPatterns(cache["message"], "(?P<webrequest>[A-Z]{3}.*?[\\d]{3})"), "insert")`,
+					`set(body, cache["message"])`,
+
+					// Set process.title to the "name" from each log record.
+					// TODO: confirm that the "name" entry in the pgAdmin logs is in fact referring
```

Comment:
Looks like this always going to say either "pgadmin" or "gunicorn.access". The [docs](https://docs.python.org/3/library/logging.html#logrecord-attributes) define name as "Name of the logger used to log the call."

Reply:
Okay, in that case this seems to me like it should be mapped to `instrumentation_scope.name`... @cbandy do you concur?

Reply:
Agreed. If the component has an idea of named loggers, then that name goes into the scope. [This](https://opentelemetry.io/docs/specs/otel/glossary/#instrumentation-scope) and the term after it talk about it.

Reply:
Thank you for the link! I was having a hard time finding documentation on Instrumentation Scope.